var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -76,22 +76,6 @@ def url_for_version(self, version):
     # variant('cdmremote', default=False,
     #         description='Enable CDM Remote support')
 
-    # These variants control the number of dimensions (i.e. coordinates and
-    # attributes) and variables (e.g. time, entity ID, number of coordinates)
-    # that can be used in any particular NetCDF file.
-    variant(
-        'maxdims',
-        default=1024,
-        description='Defines the maximum dimensions of NetCDF files.',
-        values=is_integral
```

Comment:
You can remove the `is_integral` function definition at the top of the file as well.

Reply:
Thanks. good catch.  Removed